### PR TITLE
CBL-6146: Apply dev feedback

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
Application of feedback RE ticket: https://issues.couchbase.com/browse/CBL-6146

Also added back fullsync snippets that seemed to have perished in a merge.

### CODE REVIEW

File: modules/ROOT/pages/_partials/commons/common-document.adoc [hunk 1]
```diff
@@ -263,7 +263,7 @@ ifdef::is-android[]
 [NOTE]
 --
 Care should be taken when storing and recovering data in a document or converting that document to JSON and back. +
-Data encoding (Fleece) can result in `Long` values being converted to `Float` instead of `Double`. +
```
Comment (bmeike) [inline modules/ROOT/pages/_partials/commons/common-document.adoc:266]: I am so sorry: I just noticed that this is backwards.  Needs correction here and in the backport.  It should read:

Data encoding (Fleece) can result in `Long` values being converted to `Integer`, and `Double` values to `Float`.

**Review by bmeike — CHANGES_REQUESTED** [review]
Really sorry about this.  Totally my bad.

File: modules/ROOT/pages/_partials/commons/common-document.adoc (hunk 1 above)
Comment (ElliotFrancisHunter) [reply modules/ROOT/pages/_partials/commons/common-document.adoc:266]: Done!

**Review by bmeike — APPROVED** [review]

File: modules/ROOT/pages/_partials/commons/common-document.adoc [hunk 2]
```diff
@@ -263,7 +263,7 @@ ifdef::is-android[]
...
+Data encoding (Fleece) can result in `Long` values being converted to `Integers`, and `Double` values to `Float`. +
```
Comment (simon-dew) [inline modules/ROOT/pages/_partials/commons/common-document.adoc:266]: Should this just be 'Integer`?

```suggestion
Data encoding (Fleece) can result in `Long` values being converted to `Integer`, and `Double` values to `Float`. +
```

**Review by simon-dew — APPROVED** [review]
Just one comment; otherwise LGTM. Approved to unblock.

File: modules/ROOT/pages/_partials/commons/common-document.adoc (hunk 2 above)
Comment (ElliotFrancisHunter) [reply modules/ROOT/pages/_partials/commons/common-document.adoc:266]: Added to post ga tidy

**Pull request merged**